aten/src/ATen/native/native_functions.yaml

```diff
@@ -146,6 +146,11 @@
 - func: argmin(Tensor self) -> Tensor
 - func: _argmin(Tensor self, int64_t dim, bool keepdim=false) -> Tensor
 
+# The actual implementations live in Declarations.cwrap. These are just to
+# provide default values for storage_offset=self.storage_offset()
+- func: as_strided(Tensor self, IntList size, IntList stride) -> Tensor
```

Comment:
Previously we had a default value of `-1` in `Declarations.cwrap`. It makes computing the actual offset value troublesome in backward.